Makefile

```diff
@@ -100,6 +105,12 @@ clean-log:
 	@echo "Cleaning up logs..."
 	@rm -rf log/
 
+clean-cache:
```

Comment:
I think `clean-test` is better. It' just suggestion.

Reply:
Approved!